packages/ng/popover/src/lib/trigger/popover-trigger.model.ts

```diff
@@ -199,8 +199,10 @@ export abstract class ALuPopoverTrigger<TPanel extends ILuPopoverPanel = ILuPopo
 
 	/** Closes the popover. */
 	closePopover(): void {
-		if (this._overlayRef && this._overlayRef.hasAttached()) {
-			this._overlayRef.detach();
+		if (this._overlayRef) {
```

Comment:
should we call `emitClose` when we have an unattached overlayRef ?

Reply:
> Maybe there is a way to call emitClose sooner (when the "real" close occurs with the scrolling)

I would prefer that.

In the `_createOverlay` method, we could use something like `this._overlayRef.detachments().subscribe(...)` to detect popover closing.
But with this, we have to make sure that close event is not emitted twice in the "normal" use-case.

Reply:
@GuillaumeNury i've tried to take advantage of `_overlayRef.detachments`
See my last proposal